ansible/roles/monasca/templates/monasca-thresh/thresh-config.yml.j2

```diff
@@ -0,0 +1,128 @@
+metricSpoutThreads: 2
+metricSpoutTasks: 2
+
+#statsdConfig:
+#  host: "127.0.0.1"
+#  port: %MONASCA_STATSD_PORT%
+#  prefix: monasca.storm.
+#  dimensions: !!map
+#    service : monitoring
+#    component : storm
+
+
+metricSpoutConfig:
+  kafkaConsumerConfiguration:
+  # See http://kafka.apache.org/documentation.html#api for semantics and defaults.
+    topic: "{{ monasca_metrics_topic }}"
+    numThreads: 1
+    groupId: "thresh-metric"
+    zookeeperConnect: "{{ monasca_zookeeper_servers }}"
+    consumerId: 1
+    socketTimeoutMs: 30000
+    socketReceiveBufferBytes : 65536
+    fetchMessageMaxBytes: 1048576
+    autoCommitEnable: true
+    autoCommitIntervalMs: 60000
+    queuedMaxMessageChunks: 10
+    rebalanceMaxRetries: 4
+    fetchMinBytes:  1
+    fetchWaitMaxMs:  100
+    rebalanceBackoffMs: 2000
+    refreshLeaderBackoffMs: 200
+    autoOffsetReset: largest
+    consumerTimeoutMs:  -1
+    clientId : 1
+    zookeeperSessionTimeoutMs : 60000
+    zookeeperConnectionTimeoutMs : 60000
+    zookeeperSyncTimeMs: 2000
+
+
+eventSpoutConfig:
+  kafkaConsumerConfiguration:
+  # See http://kafka.apache.org/documentation.html#api for semantics and defaults.
+    topic: "{{ monasca_events_topic }}"
+    numThreads: 1
+    groupId: "thresh-event"
+    zookeeperConnect: "{{ monasca_zookeeper_servers }}"
+    consumerId: 1
+    socketTimeoutMs: 30000
+    socketReceiveBufferBytes : 65536
+    fetchMessageMaxBytes: 1048576
+    autoCommitEnable: true
+    autoCommitIntervalMs: 60000
+    queuedMaxMessageChunks: 10
+    rebalanceMaxRetries: 4
+    fetchMinBytes:  1
+    fetchWaitMaxMs:  100
+    rebalanceBackoffMs: 2000
+    refreshLeaderBackoffMs: 200
+    autoOffsetReset: largest
+    consumerTimeoutMs:  -1
+    clientId : 1
+    zookeeperSessionTimeoutMs : 60000
+    zookeeperConnectionTimeoutMs : 60000
+    zookeeperSyncTimeMs: 2000
+
+
+kafkaProducerConfig:
+  # See http://kafka.apache.org/documentation.html#api for semantics and defaults.
+  topic: "{{ monasca_alarm_state_transistions_topic }}"
+  metadataBrokerList: "{{ monasca_kafka_servers }}"
+  serializerClass: kafka.serializer.StringEncoder
+  partitionerClass:
+  requestRequiredAcks: 1
+  requestTimeoutMs: 10000
+  producerType: sync
+  keySerializerClass:
+  compressionCodec: none
+  compressedTopics:
+  messageSendMaxRetries: 3
+  retryBackoffMs: 100
+  topicMetadataRefreshIntervalMs: 600000
+  queueBufferingMaxMs: 5000
+  queueBufferingMaxMessages: 10000
+  queueEnqueueTimeoutMs: -1
+  batchNumMessages: 200
+  sendBufferBytes: 102400
+  clientId : Threshold_Engine
+
+
+sporadicMetricNamespaces:
+  - foo
```

Comment:
erm

Reply:
Removed  - it's part of the default config!